client/packages/purchasing/src/purchase_order/ListView/OutstandingLines/OutstandingLines.tsx

```diff
@@ -44,115 +40,98 @@ const OutstandingLinesList = () => {
-        sortable: false,
+        header: t('label.item-name'),
+        accessorKey: 'itemName',
+        accessorFn: row => row.item.name,
+        enableSorting: true,
       },
       {
-        key: 'confirmedDatetime',
-        label: 'label.purchase-order-confirmed',
-        formatter: dateString =>
-          dateString ? localisedDate((dateString as string) || '') : '',
-        accessor: ({ rowData }) => rowData?.purchaseOrder?.confirmedDatetime,
-        sortable: false,
+        header: t('label.purchase-order-confirmed'),
+        accessorKey: 'purchaseOrder.confirmedDatetime',
+        columnType: ColumnType.Date,
       },
       {
-        key: 'expectedDeliveryDate',
-        label: 'label.expected-delivery-date',
-        format: ColumnFormat.Date,
-        sortable: false,
-        width: 150,
+        header: t('label.expected-delivery-date'),
+        accessorKey: 'purchaseOrder.expectedDeliveryDate',
+        columnType: ColumnType.Date,
       },
       {
-        key: 'adjustedNumberOfUnits',
-        label: 'label.adjusted-units-expected',
-        Cell: NumberCell,
-        sortable: false,
-        width: 150,
+        header: t('label.adjusted-units-expected'),
+        accessorKey: 'adjustedNumberOfUnits',
+        columnType: ColumnType.Number,
       },
       {
-        key: 'receivedNumberOfUnits',
-        label: 'label.received-units',
-        Cell: NumberCell,
-        sortable: false,
+        header: t('label.received-units'),
+        accessorKey: 'receivedNumberOfUnits',
+        columnType: ColumnType.Number,
       },
       {
-        key: 'outstandingQuantity',
-        label: 'label.outstanding-units',
-        Cell: NumberCell,
-        accessor: ({ rowData }) => {
-          const adjusted = rowData?.adjustedNumberOfUnits ?? 0;
-          const received = rowData?.receivedNumberOfUnits ?? 0;
+        header: t('label.outstanding-units'),
+        accessorKey: 'outstandingQuantity',
+        columnType: ColumnType.Number,
+        accessorFn: row => {
+          const adjusted = row?.adjustedNumberOfUnits ?? 0;
+          const received = row?.receivedNumberOfUnits ?? 0;
           return adjusted - received;
         },
-        sortable: false,
       },
     ],
-    { onChangeSortBy: updateSortQuery, sortBy },
-    [sortBy]
+    []
   );
 
+  const { table } = usePaginatedMaterialTable<PurchaseOrderLineFragment>({
+    tableId: 'outstanding-purchase-order-lines-list',
+    isLoading: isLoading,
+    isError,
+    onRowClick: row =>
+      navigate(
+        RouteBuilder.create(AppRoute.Replenishment)
+          .addPart(AppRoute.PurchaseOrder)
+          .addPart(row.purchaseOrder?.id ?? '')
+          .build()
+      ),
+    columns: mrtColumns,
+    data: data?.nodes,
+    totalCount: data?.totalCount ?? 0,
+    initialSort: { key: 'invoiceNumber', dir: 'desc' },
+    noDataElement: (
+      <NothingHere body={t('message.no-outstanding-purchase-order-lines')} />
+    ),
+  });
```

Comment:
There are no actions for selected row here, so add `enableRowSelection: false` to hide the selection column